rest-client-base/src/main/java/com/wultra/core/rest/client/base/RestClientConfiguration.java

```diff
@@ -47,9 +50,11 @@ public RestClientConfiguration() {
 
     // HTTP connection timeout
     private Integer connectionTimeout = 5000;
+    private Duration responseTimeout;
 
     // TLS certificate settings
     private boolean acceptInvalidSslCertificate = false;
+    private Duration handshakeTimeout;
```

Comment:
Shouldn't we document the default value here?

`SslProvider:335`
```static final long DEFAULT_SSL_HANDSHAKE_TIMEOUT = Long.parseLong(System.getProperty("reactor.netty.tcp.sslHandshakeTimeout", "10000"));```

Reply:
Default value is documented at setter.